pkg/sriov/poolconfig_test.go

```diff
@@ -240,6 +240,40 @@ func TestWithMaxUnavailable(t *testing.T) {
 	}
 }
 
+func TestWithRdmaMode(t *testing.T) {
```

Comment:
another nit, but this could be updated to match the capitalization of the function
```suggestion
func TestWithRDMAMode(t *testing.T) {
```

Reply:
done. ptal